apps/site/src/components/pages/hub/block-data-container.tsx

```diff
@@ -30,6 +31,22 @@ import { BlockVariantsTabs } from "./block-data-container/block-variants-tabs";
 import { SandboxedBlockDemo } from "./block-data-container/sandboxed-block-demo";
 import { BlockExampleGraph, BlockSchema } from "./hub-utils";
 
+const intervalForAllTime =
```

Comment:
Any further thoughts on sharing this somewhere? `@blockprotocol/graph/internal`?

Reply:
It's only used in one place currently so I'm not sure that's necessary 
<img width="984" alt="image" src="https://user-images.githubusercontent.com/25749103/229092865-565b9009-0dbb-4c5e-8575-f4386e541126.png">